test.md

```diff
@@ -0,0 +1 @@
+This file should be ignored by git
```

Comment:
- [ ] This file should be ignored by git. You could do this by adding it in the `.gitignore` directory. This is important because we don't want to mistakenly commit unwanted files in our repository and we also have to keep our repository professional. [You can read more about it here.](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/documentation/articles/professional_repo_rules.md)